ci/prow/e2e-crd-variable

```diff
@@ -17,7 +16,7 @@ echo "Wait until the KMMO Deployment is Available..."
 oc wait --for condition=Available --timeout 1m deployments.apps -n kmm-operator-system kmm-operator-controller-manager
 
 echo "Check that the kmm_ci_a module is not loaded on the node..."
-if check_kmm_kmod; then
+if . ./ci/prow/check-kmm-kmod; then
```

Comment:
What does that command do?

Reply:
source is a synonym for dot/period '.' in bash. So it's just like source command

Reply:
Why don't we use `source` in that case? It is more explicit.

Reply:
shorter?

Reply:
I see. Thanks for the explanation. The way I am seeing it (which is usually what we do in `go` as well):
* `ci/prow/check-kmm-kmod` should only contain the function definition (not run it)
* each test using that function should `source` the file at the begining of the script (same as we do for importing packages in `go`, `python`, etc...
* each test that requires `check_kmm_kmod` function should call it explicitely.

WDYT?

Reply:
> I see. Thanks for the explanation. The way I am seeing it (which is usually what we do in `go` as well):
> 
> * `ci/prow/check-kmm-kmod` should only contain the function definition (not run it)
> * each test using that function should `source` (explicitely) the file at the begining of the script (same as we do for importing packages in `go`, `python`, etc...
> * each test that requires `check_kmm_kmod` function should call it explicitely.
> 
> WDYT?

We sourced the function at the beginning of each script before but as I explain in commit, source is lost when you call the function from a new session (bash -c), hence the change.